fv/Dockerfile.test.amd64

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2015-2019 Tigera, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# NOTE: This Dockerfile should be kept in-sync with the one in calico/node.
+# This ensures that testing of Felix in this repository is done in the same
+# userspace environment as it will be deployed in calico/node.
+FROM calico/felix:latest
+LABEL maintainer="Shaun Crampton <shaun@tigera.io>"
+
+
+# Install remaining runtime deps required for felix from the global repository
+RUN apt-get update && apt-get install -y \
```

Comment:
How does this perform?  Just wondering if it adds quite a few seconds to the build :thinking: